### ISSUE TITLE
BfA update

### ISSUE THREAD

**Issue by ThiefMaster** [eldir/Altoholic#2]
Hey, since the original addon still appears to be abandoned, is there any chance you are going to update your fork for BfA?

**Comment by eldir** [comment]
Hey mate, the code would require a major overhaul. There are tons of things even from WoD left in it. I don't think I will tackle that task, I was hoping that the original dev will resurface come BfA.

**Comment by eldir** [comment]
I'd be happy to work in a group on this, but I just don't have the time to do it alone in my spare time.

**Comment by eldir** [comment]
I'd be happy to work in a group on this, but I just don't have the time to do it alone in my spare time.

**Comment by sk4p** [comment]
I've begun working on it myself and have a repo for it.  I saw Thaoky did make a post to WowInterface that he(?) is going to look at the code soon but has a lot of RL stuff going on.

**Comment by eldir** [comment]
@sk4p thanks a lot for the tip, I had not seen the post yet! I've tried contacting Thaoky through a PM there to see if he wants help with the necessary dev work.

**Comment by sk4p** [comment]
You bet.  I've done likewise.  I dunno if he's willing to work with collaborators.  If he's not, since he licensed the addon as All Rights Reserved, I may just take a deep breath and start from scratch for my particular alting needs, if it's going to run into months before he updates (which it sounds like may be the case, given what he said in the post about RL stuff going on).

If I decide to take such a plunge, I'd definitely put it up under a more open license like GLPL or some flavor of CC, because I don't want the next person to get put in this position.  Interested in contributing if I do?

**Comment by eldir** [comment]
I've gotten a reply from Thaoky that he's already reworking parts of the Alcoholic and pushed an update over the weekend.

**Comment by sk4p** [comment]
Yeah, the update fixes a bunch of the random LUA errors.  It's not done (by his own admission) and he's still fixing in.  I was pleasantly surprised to see him on it so quickly.

So I'm not going to tackle a rewrite myself at this time.  I am interested in making life easier for folks so I'm casting about for other addons people are wishing existed. :) Cheers!

**Comment by eldir** [comment]
I was considering something aimed at farming/grinding on alts with a simple input interface where you link a quest or raid lockout and it would then track that with a daily/weekly lockout across all alts until you delete it (rather than the other way around where you need to log into an alt to have addons pick up that you're on a specific quest)

**Comment by sk4p** [comment]
That sounds like an addon which already exists called [SavedInstances](https://www.wowace.com/projects/saved_instances).  Would that do what you want?

Well, let me qualify that: it tracks your instance and raid lockouts but doesn't know anything about quests.  Can you describe the situation with an example?  I've not had enough tea yet this morning. :)

**Comment by sk4p** [comment]
I think I can write something like that and I (with my 17 level 110s) would find it useful.  Hmmm.

**Comment by eldir** [comment]
yeah essentially, but not just lockouts also certain quests, daily profession items if any, basically attach anything that's linkable to a daily or weekly lockout timer

**Comment by sk4p** [comment]
Sure.  I think, mind you, that Altoholic includes some stuff like "Hexweave Cloth is now ready on Tailor" sort of messages.  But I'm envisioning something not just linkable but even free-form, like you could add a text item to the checklist like "Run around Orgrimmar shouting 'The End Is Near'" and add it to your demon hunter and warlock and then when you log into anyone, you see a "to do list" advising you of it (again, like Altoholic warns you that mail is expiring on some alts).

**Comment by sk4p** [comment]
I just learned about [Executive Assistant](https://www.curseforge.com/wow/addons/exec_assist).  I wonder if it provides what you and I are each looking for.

**Comment by Thaendril** [comment]
I was about to mention Executive to you guys see if that was closer on the mark of what you were pitching there.   That said, something like that embedded into Alto without needing another mod into the fold would be nice.   

Truth be told though if it wasn't put here and Executive isn't exactly whats needed I'm all for the development of new and improved things.

**Comment by sk4p** [comment]
One helpful tip regarding Thaoky's new Altoholic: Don't forget to delete your pre-8.0 SavedVariables files.  Even with his new code, I was still getting some lua errors and weird behavior.  When I nuked all the SavedVariables and started fresh, boom, suddenly everything works as advertised.  (He doesn't have professions working just yet, but now all the basic account summary, reputation, and inventory search stuff works great for me.)
